### ISSUE TITLE
The bitlang/cobol plugin repository seems to have been emptied

### ISSUE THREAD

**Issue by ericwill** [eclipse-che/che#18358]
There's a few things wrong with the `bitlang/cobol` plugin at the moment:
* The upstream repository is empty, except for a few images: https://github.com/spgennard/vscode_cobol_extension
* The upstream repository no longer has any tags/release information
* I am pretty sure the folder name is wrong

@filipkroupa you were the last one to make any contribution here, do you know what is going on with this plugin? Do you still need it?

**Comment by filipkroupa** [comment]
@ericwill It seems the author of the extension moved the repository here [https://github.com/spgennard/vscode_cobol](https://github.com/spgennard/vscode_cobol). Anyway we were the original publisher of this plugin into Che plugin registry, since we used this plugin together with our plugin `broadcommfd/cobol-language-support`. But since then, we implemented the functionality provided by `bitlang/cobol` in our plugin and we no longer need the extra plugin. It was removed from our devfile `che4z` and we don't use it.
